src/include/migraphx/matcher.hpp

```diff
@@ -567,6 +567,25 @@ inline auto has_value(T x, float tolerance = 1e-6)
     });
 }
 
+template <class T>
+inline auto has_value_multi(T x, float tolerance = 1e-6)
```

Comment:
Is a new matcher necessary? Can't we just rename this to `has_value`?

Reply:
I believe so. Initially I checked bert_mrpc, and the way we had written `has_value` was limited to literals of size 1. In the onnx file, it seems they had already broadcasted those values, so the matcher was incorrect.